pkg/tfshim/sdk-v2/diagnostics.go

```diff
@@ -14,7 +14,11 @@ func warningsAndErrors(diags diag.Diagnostics) ([]string, []error) {
 		case diag.Error:
 			errors = append(errors, fromV2Diag(d))
 		case diag.Warning:
-			warnings = append(warnings, d.Summary)
+			details := d.Summary
```

Comment:
Can we add a comment citing an example of what might be in `Summary` vs `Detail` and why we would would prefer the latter?

Reply:
So the fix kinda denotes that right? I mean, the summary doesn't include the parameter name, the detail does :)